pkgs/development/compilers/ghc/head.nix

```diff
@@ -54,10 +68,11 @@ let
     EXTRA_CC_OPTS += -std=gnu99
   '';
 
+
```

Comment:
blank line wasn't added in 8.4.1. In general it's nice to diff the different GHCs so they don't unnecessarily drift apart. We really should deduplicate soon...